CMakeLists.txt

```diff
@@ -16,13 +17,28 @@ add_compile_options(
         -Wlogical-op
 )
 include_directories(${CMAKE_SOURCE_DIR}/src)
+include_directories(${CMAKE_SOURCE_DIR}/src/utils)
 include_directories(${CMAKE_SOURCE_DIR}/vendor/spdlog/include/)
 
-
 set(CMAKE_CXX_STANDARD 17)
 
 add_library(bald_engine library.cpp library.h)
 
-
 set_target_properties(bald_engine PROPERTIES COTIRE_CXX_PREFIX_HEADER_INIT "src/pch.h")
-cotire(bald_engine)
\ No newline at end of file
+cotire(bald_engine)
+
+enable_testing()
+#file(GLOB TEST_SRC_FILES ${PROJECT_SOURCE_DIR}/tests/*.cpp)
+
+message("-- TEST EXECUTALES: " ${TEST_SRC_FILES})
+#add_executable(runUnitTests ${TEST_SRC_FILES})
+
+add_executable(runUnitTests
```

Comment:
name of the test should be somthing like file_menager_test

Reply:
done